assert/assertions.go

```diff
@@ -2202,11 +2202,17 @@ func ErrorAs(t TestingT, err error, target interface{}, msgAndArgs ...interface{
 		return true
 	}
 
+	expectedText := reflect.ValueOf(target).Elem().Type().String()
```

Comment:
Use `reflect.TypeOf(target).Elem().String()`

Reply:
I had copy pasted existing code.

I updated everything with #1740

- #1740

I close this thread, please provide feedbacks in other PR